src/migrations/20201008104340-create-user.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = {
```

Comment:
I suggest you make use of ES6 import/export modules

Reply:
I think soo he can use`export default {.....}`

Reply:
yeah, I did use export default but it threw a big amount of errors